README.md

```diff
@@ -53,7 +57,7 @@ weather === "sunny";
 
 What are the values of these expressions?  Explain your answers.
 
-<!-- Replace this comment with your answer -->
+"=" is the assign symbol, so line 2 assign string "sunny" to variable weather. "===" is the strictly equal symbol, it compares the type and value of both sides and returns boolean values true or false. In this case, line 3 evaluates TRUE.
```

Comment:
This is true, but `weather = "sunny"` also evaluates to a value - specifically, "sunny". `var weather;` evaluates to `undefined`.